src/components/Pages/Auth/FormAuth.tsx

```diff
@@ -28,6 +28,9 @@ const AuthForm: FC = () => {
       <Button className="mt-3 w-full" type="submit">
         Next
       </Button>
+      {/* <Button className="mt-3 w-full" type="submit">
+        Next
+      </Button> */}
```

Comment:
You can remove this beacuse it's the same code as you've used above

Reply:
Ok